stack.yaml

```diff
@@ -6,7 +6,9 @@ packages:
 
 extra-deps:
   - alex-3.2.5
+  - binary-instances-1.0.1
```

Comment:
Which instances do we need from this package?

Reply:
the `github` package claimed to need this 🤷

Reply:
Ah, `github` is coming from Hackage. Gotcha 👍